src/screens/generation/generation.vue

```diff
@@ -83,7 +83,13 @@
 
     <div class="block-micros">
       <h2 class="title">
-        <span class="number">{{ generation.micros.length }}</span> Micro Block(s)
+        <span class="number">{{ generation.micros.length }}</span>
+        <span v-if="generation.micros.length < 1">
+          Micro Blocks
+        </span>
+        <span v-else>
+          Micro Block
+        </span>
```

Comment:
It looks too cluttered, you can use ternary operator:
```vue
<span class="number">{{ generation.micros.length }}</span>
Micro Block{{ generation.micros.length !== 1 ? 's' : '' }}
```

Reply:
good point! I will change this, and also add it for the transaction(s) title